.travis.yml

```diff
@@ -39,22 +37,65 @@ jobs:
         - ./vendor/bin/phpunit
 
     - stage: Test
+      php: 7.2
       env: COVERAGE
       before_script:
         - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{.disabled,}
         - if [[ ! $(php -m | grep -si xdebug) ]]; then echo "xdebug required for coverage"; exit 1; fi
       script:
-        - ./phpunit --build-all --coverage-clover clover.xml
+        - ./bin/console --env=test build-all
+        - ./vendor/bin/phpunit --coverage-clover clover.xml
       after_script:
         - wget https://scrutinizer-ci.com/ocular.phar
         - php ocular.phar code-coverage:upload --format=php-clover clover.xml
 
     - stage: Code Quality
+      php: 7.2
       env: CODING_STANDARDS
       install: travis_retry composer install --prefer-dist
       script: ./vendor/bin/phpcs
 
     - stage: Code Quality
+      php: 7.2
       env: STATIC_ANALYSIS
       install: travis_retry composer install --prefer-dist
       script: ./vendor/bin/phpstan analyse
+
+    # PHP 7.4
+    - stage: Test
+      php: 7.4
+      env: DEPENDENCIES=low
+      install:
+        - rm composer.lock
+        - travis_retry composer update --prefer-dist --prefer-lowest
+        - travis_retry yarn install
+      script:
+        - ./vendor/bin/phpunit
+
+    - stage: Test
+      php: 7.4
+      env: COVERAGE
+      before_script:
+        - mv ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini{.disabled,}
+        - if [[ ! $(php -m | grep -si xdebug) ]]; then echo "xdebug required for coverage"; exit 1; fi
+      script:
+        - ./bin/console --env=test build-all
+        - ./vendor/bin/phpunit --coverage-clover clover.xml
+      after_script:
+        - wget https://scrutinizer-ci.com/ocular.phar
+        - php ocular.phar code-coverage:upload --format=php-clover clover.xml
+
+    - stage: Code Quality
+      php: 7.4
+      env: CODING_STANDARDS
+      install: travis_retry composer install --prefer-dist
+      script: ./vendor/bin/phpcs
+
+    - stage: Code Quality
+      php: 7.4
+      env: STATIC_ANALYSIS
+      install: travis_retry composer install --prefer-dist
+      script: ./vendor/bin/phpstan analyse
+
+  allow_failures:
+    - php: 7.4
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.